### ISSUE TITLE
Spotify Integration

### ISSUE THREAD

**Issue by AltraMage** [OlegWock/anori#51]
I would like to see a way to control Spotify from a new tab, I haven't worked with the spotify API before, but would love to help on this project as a whole if you're open to that

**Comment by OlegWock** [comment]
That sounds good. Do you mean controlling desktop spotify (or spotify playing on another device) or one open in browser?

**Comment by xhos** [comment]
Kinda late for this, but I think controlling the spotify desktop app would be more relevant, since (I may be wrong) more people use it then the web version. Really hoping to see this feature in the future :)

**Comment by OlegWock** [comment]
Yeah, I agree that it makes most sense. But it seems this will require either
* Creating proper Spotify API APP, setup OAuth and use public Spotify API to control player
* Reverse engineer Spotify private API and use it to control player 

I think both are doable, with its own pros and cons. However, both require a lot of work 😅
I currently don't have enough free time to take on such big tasks, so if someone reading this want to make a PR, you're welcome

**Comment by OlegWock** [comment]
Removing 'Pull requests are welcome' label for now due to this https://github.com/OlegWock/anori/issues/47#issuecomment-1627816823 and this https://github.com/OlegWock/anori/issues/47#issuecomment-1627817044

**Comment by OlegWock** [comment]
Hello, I'm doing issues cleanup in this repo. I'll be closing this issue because it's pretty old and I don't think it will get implemented
